### PR TITLE
Allow `:dir_browser` `htdocs_path` to be specified in config.

### PR DESCRIPTION
**Pull request by dom111**
Addresses #191 .

I haven't updated any docs or anything, didn't want to jump the gun!

### CODE REVIEW

File: wsgidav/dir_browser/_dir_browser.py [hunk 1]
```diff
@@ -45,7 +45,11 @@ class WsgiDavDirBrowser(BaseMiddleware):
 
     def __init__(self, wsgidav_app, next_app, config):
         super(WsgiDavDirBrowser, self).__init__(wsgidav_app, next_app, config)
-        self.htdocs_path = os.path.join(os.path.dirname(__file__), "htdocs")
+        self.dir_config = config.get("dir_browser", {})
+        self.htdocs_path = os.path.realpath(self.dir_config.get('htdocs_path', os.path.join(os.path.dirname(__file__), "htdocs")))
+
+        if not os.path.exists(self.htdocs_path):
```
Comment (mar10) [inline wsgidav/dir_browser/_dir_browser.py:51]: `.isdir(...)` would be even more specific

**Review by mar10 — CHANGES_REQUESTED** [review]
LGTM.
Please update the docs also, where appropriate.
Thanks for contributing!

File: wsgidav/dir_browser/_dir_browser.py [hunk 2]
```diff
@@ -45,7 +45,11 @@ class WsgiDavDirBrowser(BaseMiddleware):
 
     def __init__(self, wsgidav_app, next_app, config):
         super(WsgiDavDirBrowser, self).__init__(wsgidav_app, next_app, config)
-        self.htdocs_path = os.path.join(os.path.dirname(__file__), "htdocs")
+        self.dir_config = config.get("dir_browser", {})
+        self.htdocs_path = os.path.realpath(self.dir_config.get('htdocs_path', os.path.join(os.path.dirname(__file__), "htdocs")))
```
[the remark this answers is not in the document]
Comment (dom111) [reply wsgidav/dir_browser/_dir_browser.py]: Fixed!

File: wsgidav/dir_browser/_dir_browser.py (hunk 1 above)
Comment (dom111) [reply wsgidav/dir_browser/_dir_browser.py]: Changed!

**Comment by dom111** [comment]
I've added docs to where I think makes sense, but please let me know if there's anywhere else relevant in your opinion I should add detail!

**Pull request merged**